src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/actions.tsx

```diff
@@ -76,11 +79,7 @@ export async function updateOnerepProfile(
 
 export async function getAllProfileScans(onerepProfileId: number) {
   const session = await getServerSession();
-  if (
-    !session?.user?.email ||
-    !isAdmin(session.user.email) ||
-    process.env.APP_ENV === "production"
-  ) {
+  if (!session?.user?.email || !isAdmin(session.user.email)) {
```

Comment:
Since this potentially exposes PII as well (I think?), maybe we should do a check that this profile ID is associated with a Mozilla email address as well?

Reply:
I was thinking about this as well. Right now, we are not calling this action without checking the email address of the subscriber which is checked before. I’ve added a check to the function directly as an additional safeguard in [7b598da](https://github.com/mozilla/blurts-server/pull/5850/commits/7b598dab3a98bfcab7e7fe0d43a4186da03ede98).

Reply:
Right, but keep in mind that "calling the action" is an API call that the user can simulate without doing the email check beforehand.